client/src/app/components/login/login.component.css

```diff
@@ -0,0 +1,101 @@
+.background{
+    background-image: url(../../../assets/images/park.jpg); background-repeat: no-repeat;
+
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
```

Comment:
Angular will add any necessary prefixes automatically, so there's no need to write them explicitly in the CSS files (but it's OK if you do).